tests/test_migration.py

```diff
@@ -2,15 +2,17 @@
 #       Use another copy of the strategy to simulate the migration
 #       Show that nothing is lost!
 
+import pytest
+
 
 def test_migration(token, vault, strategy, amount, Strategy, strategist, gov):
     # Deposit to the vault and harvest
     token.approve(vault.address, amount, {"from": gov})
     vault.deposit(amount, {"from": gov})
     strategy.harvest()
-    assert token.balanceOf(strategy.address) == amount
+    assert pytest.approx(strategy.estimatedTotalAssets(), rel=1e-5) == amount
```

Comment:
can you make these constants imported from the `conftest.py` (or available as a session fixture maybe)

Reply:
done